applications/StructuralMechanicsApplication/custom_processes/set_automated_initial_variable_process.cpp

```diff
@@ -0,0 +1,124 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Luis Antônio Gonçalves Junior
```

Comment:
I don't want to look "anti-Portuguese", but unfortunately to avoid problem with some compilers we should avoid special characters 
```suggestion
//  Main authors:    Luis Antonio Goncalves Junior
```
Sorry

Reply:
No worries. I didn't know the comments would affect the compilation process.